README.md

```diff
@@ -3,37 +3,20 @@
 [![CI Status](https://github.com/coralnet/pyspacer/actions/workflows/python-app.yml/badge.svg)](https://github.com/coralnet/pyspacer/actions/workflows/python-app.yml)
 [![PyPI version](https://badge.fury.io/py/pyspacer.svg)](https://badge.fury.io/py/pyspacer)
 
-This repository provide utilities to extract features from random point 
-locations in images and then training classifiers over those features.
+This repository provides utilities to extract features from random point 
+locations in images and then train classifiers over those features.
 It is used in the vision backend of `https://github.com/coralnet/coralnet`.
 
 Spacer currently supports python >=3.8.
 
-### Overview
-Spacer executes tasks as defined in messages. The messages types are defined
-in `messages.py` and the tasks in `tasks.py`. We also define several data-types
-in `data_classes.py` which define input and output types. 
-
-Refer to the unit-test in `test_tasks.py` for examples on how to create tasks.
-
-Tasks can be executed directly by calling the methods in tasks.py. 
-However, spacer also supports an interface with AWS Batch 
-handled by `env_job()` in `mailman.py`. 
-
-Spacer supports four storage types: `s3`, `filesystem`, `memory` and `url`.
- Refer to `storage.py` for details. The Memory storage is mostly used for 
- testing, and the `url` storage is read only.
-
-Also take a look at `config.py` for settings and configuration. 
-
-### Installation
+## Installation
 
 The spacer repo can be installed in three ways.
-* Using Docker -- the only option that supports Caffe.
+* Pip install -- for integration with other Python projects.
 * Local clone -- ideal for testing and development.
-* Using pip install -- for integration in other code-bases.
+* From Dockerfile -- the only option that supports Caffe, which is used for the legacy feature-extractor.
 
-#### Config
+### Config
```

Comment:
It might be good to write out somewhere in this section (perhaps at the end) the checking order, i.e. which sources trump others if multiple sources are defined.

Reply:
Good idea, will add that.

Reply:
See commit 41c6724